tests/tests/test_general.py

```diff
@@ -206,3 +212,116 @@ def test_named_logger_doesnt_propagate(tmp_path, capsys):
     assert "PQ&*" not in captured.out, (
         "logger writing to stdout through root handler"
     )
+
+
+def test_python_version_header_absent(tmp_path):
+    """Python version section does not exist if logger
+    is created with the parameter write_python_version as False.
+    """
+    ver_header = f"{lateral_separator}  PYTHON VERSION  {lateral_separator}\n"
```

Comment:
Nitpicking, but this could be a [pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) to reduce duplication

Reply:
They can take some getting used to, but they're very helpful to create "things" to be reused.